### PR TITLE
Add simple contributing file

### PR DESCRIPTION
**Pull request by asdine**
Because Genji is starting to get more and more contributions, I think this is the right time to add a CONTRIBUTING.md file.
The purpose of this file is to establish some clear guidelines on how we want to work together and to make sure any important change is discussed before anyone starts spending energy on it.
As a maintainer, my role is to structure this project, and obviously not to find all solutions myself.
This PR is my attempt to make sure contributions are coherent, and make sure anyone (not only maintainers) can give their opinion.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+## Reporting a bug
+
+We use Github issues to track bugs. Report a bug by [opening an issue](https://github.com/genjidb/genji/issues/new), it's that easy!
+
+Make sure you give enough context for us to reproduce the bug:
+
+- The version of Genji you are using
+- The observed behavior
+- The expected behavior
+- Ideally, some code like a main file or a Go [playground](https://play.golang.org/) link
```
Comment (tie) [inline CONTRIBUTING.md:24]: We can also create [issue templates](https://github.blog/2016-02-17-issue-and-pull-request-templates) (bug report, feature request, etc). See https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates

File: README.md [hunk 2]
```diff
@@ -290,10 +290,10 @@ genji --badger pathToData
...
-- Feedback
-- Feature ideas
-- Bug reports
-- Pull Requests
-- Anything that comes to mind to improve Genji!
+See [CONTRIBUTING.md](CONTRIBUTING.md).
+
+Thank you, [contributors]!
+
+[contributors]: https://github.com/genjidb/genji/graphs/contributors
```
Comment (tie) [inline README.md:297]: ❤️

File: CONTRIBUTING.md (hunk 1 above)
Comment (tzzed) [reply CONTRIBUTING.md:24]: Very usefull !

Comment (asdine) [reply CONTRIBUTING.md]: Yes definitely, I opened an issue 👍🏼

**Pull request merged**